src/main/webapp/app/overview/course-discussion/course-discussion.component.html

```diff
@@ -142,12 +142,26 @@
             {{ totalItems === 0 ? ('artemisApp.metis.noPosts' | artemisTranslate) : ('artemisApp.metis.postNumberInformation' | artemisTranslate: { number: totalItems }) }}
         </div>
         <!-- list of posts -->
-        <div id="scrollableDiv" class="col-12" infinite-scroll (scrolled)="fetchNextPage()">
-            <jhi-posting-thread *ngFor="let post of posts; trackBy: postsTrackByFn" [post]="post" [showAnswers]="posts.length === 1"></jhi-posting-thread>
-        </div>
-        <!-- spinner while loading posts -->
-        <div class="text-center">
-            <div *ngIf="isLoading" class="spinner-border mt-3" role="status"></div>
+        <div class="col-12">
+            <virtual-scroll
```

Comment:
Can we also use this in the `messages` feature? I see it is only added in the course discussion section

Reply:
Not at the moment. The approach there is totally different with reversed displaying of items.